src/guides/v2.3/config-guide/multi-site/ms_nginx.md

```diff
@@ -126,6 +126,9 @@ To create multiple virtual hosts:
    }
    ```
 
+   {:.bs-callout-info}
+   The `map` directive might not work correctly, because it runs only once for the same variable. The variables are evaluated only when they are used, and as a result, the mere declaration even of a large number of `map` variables does not add any extra costs to request processing. Instead of using a `map` directive, the following example sets a variable directly, like `set $MAGE_RUN_TYPE store; set $MAGE_RUN_CODE fr;`
```

Comment:
Not understanding the relationship between the directive running only once for the same variable, and declaring large number of `map` variables are evaluated only when used ...and does not add any extra processing costs.
```suggestion
   The `map` directive might not work correctly because it runs only once for the same variable. The variables are evaluated only when they are used, and as a result, even the declaration of a large number of `map` variables does not add any extra costs to request processing. Instead of using a `map` directive, the following example sets a variable directly, like `set $MAGE_RUN_TYPE store; set $MAGE_RUN_CODE fr;`
```